arm/Microsoft.ContainerService/managedClusters/deploy.bicep

```diff
@@ -107,7 +107,7 @@ param aksClusterEnablePrivateCluster bool = false
 param primaryAgentPoolProfile array
 
 @description('Optional. Define one or multiple node pools')
```

Comment:
One afterthought to keep the parameter meaningful: Update the description to `Optional. Define one or more secondary/additional node pools` in both this file & the readme

Reply:
Updated the descriptions as requested in both deploy.bicep and the readme